recipes/chembl_structure_pipeline/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "chembl_structure_pipeline" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/chembl/ChEMBL_Structure_Pipeline/archive/{{ version }}.tar.gz
+  sha256: f74d5174094ed4c9d68ba2b870c54cccc7f9fc42c09fe691852ab1d6a990d551
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - rdkit >=2019.09.2
```

Comment:
Is this needed in host?

Reply:
Yes I got an error without it before.

Reply:
see https://dev.azure.com/conda-forge/84710dde-1620-425b-80d0-4cf5baca359d/_build/results?buildId=204104&view=logs&jobId=10bda42a-d420-5c28-a43f-bc4e66405e5c

Reply:
Because of https://github.com/chembl/ChEMBL_Structure_Pipeline/blob/ba3cd2ca36bfbc1103c953a39afdfe070e6f05a9/setup.py#L2